Field.h

```diff
@@ -125,4 +126,38 @@ class TextField: public Field {
         string getValue() {
             return fieldValue;
         }
+};
+
+class CharField: public Field {
+    
+    private:
+        string fieldValue;
+        int maxLength;
+        
+    public:
+        CharField(string name, int maxLen, string defaultValue="", bool primary=false, bool unique=false, bool nullable=true):
```

Comment:
Hmmm I see that. Seems there is quite a lot of my stuff that needs to be refactored as well. The `_isNull` also breaks consistency.

Okay, this is what I think we should do:
1. all private member variables get an underscore in front of them (someone using this class can't access those variables directly anyway, so I would rather they were ugly than the method parameters).
1. We avoid use of `this->` and `this.`
1. We make the method signatures as pretty as possible, meaning full length words and consistency throughout.

We may need to look through my existing code and make sure it conforms or update it if it doesn't. Don't worry about that for this task though, rather just write your class with these rules.